packages/mdctl-cli/lib/package/publish.js

```diff
@@ -0,0 +1,45 @@
+const { isSet } = require('@medable/mdctl-core-utils/values'),
+      Package = require('../../../mdctl-packages'),
+      { Registry, Cortex } = require('../package/source'),
+      publishPkg = async(name, params) => {
+        const {
+                source, registryUrl, registryProjectId, registryToken, client
+              } = params,
+              pkg = new Package(name, '.')
+
+        await pkg.evaluate()
+
+        let srcClient
+
+        if (source === 'registry') {
+          if (!isSet(registryUrl)) {
```

Comment:
why not adding this checking inside the Registry class?

Reply:
Good catch there. I actually just realized that I added that check inside Registry class already. Let me remove that check from publish.js.